xds/internal/balancer/clusterresolver/clusterresolver.go

```diff
@@ -306,6 +310,9 @@ func (b *clusterResolverBalancer) run() {
 			switch update := u.(type) {
 			case *ccUpdate:
 				b.handleClientConnUpdate(update)
+				if update.done != nil {
```

Comment:
nit: add inline comment for the purpose of this if condition

Reply:
Updated